src/components/Sidebar/Tabs/CustomizationTab/MarkSubTab.tsx

```diff
@@ -62,12 +67,7 @@ export default function MarkSubTab(props: CustomizeSubTapProps) {
         draftSpec.params[0].select = 'interval';
       }
     });
-
-    if (typeof newSpec.mark === 'object') {
-      setSelectedMark(newSpec.mark.type);
-    } else {
-      setSelectedMark(newSpec.mark);
-    }
```

Comment:
This might break going from scatter plot to boxplot or other charts that can have mark as an object.